code/API_definitions/kyc-match.yaml

```
@@ -0,0 +1,536 @@
+        - The idDocument property is missing.(`{"code": "KNOW_YOUR_CUSTOMER.ID_DOCUMENT_REQUIRED","message": "The idDocument is required to perform the properties validation"}`)
+        - The idDocument does not match the one associated to the provided phoneNumber in the OB systems.(`{"code": "KNOW_YOUR_CUSTOMER.ID_DOCUMENT_MISMATCH","message": "The idDocument needs to match the one associated with the provided phoneNumber"}`)
+      content:
+        application/json;charset=UTF-8:
+          schema:
+            $ref: '#/components/schemas/ErrorInfo'
+          examples:
+            PermissionDenied:
+              value:
+                status: 403
+                code: PERMISSION_DENIED
+                message: Client does not have sufficient permissions to perform this action
+            InvalidTokenContext:
+              value:
+                status: 403
+                code: KNOW_YOUR_CUSTOMER.INVALID_TOKEN_CONTEXT
+                message: Phone number mismatch with access token context
+            IdDocumentRequired:
+              value:
+                status: 403
+                code: KNOW_YOUR_CUSTOMER.ID_DOCUMENT_REQUIRED
+                message: The idDocument is required to perform the properties validation
+            IdDocumentMismatch:
+              value:
+                status: 403
+                code: KNOW_YOUR_CUSTOMER.ID_DOCUMENT_MISMATCH
+                message: The idDocument needs to match the one associated with the provided phoneNumber
+
+    Generic404:
+      description: |
+        Not Found error.  Error if URL is wrong / user is not found.
+      content:
+        application/json;charset=UTF-8:
+          schema:
+            $ref: '#/components/schemas/ErrorInfo'
+          examples:
+            NotFound:
+              value:
+                status: 403
+                code: NOT_FOUND
+                message: not_found_contractor/not_found
+
+    Generic500:
+      description: Server error.  Problem with MNO's server side.  Some processing error within MNO's servers.
+      content:
+        application/json;charset=UTF-8:
+          schema:
+            $ref: '#/components/schemas/ErrorInfo'
+          examples:
+            ServerError:
+              value:
+                status: 500
+                code: INTERNAL
+                message: Server error
+
+    Generic503:
+      description: Service unavailable. Typically the server is down.  Problem with MNO's server side.  Any unexpected error within MNO's servers.
+      content:
+        application/json;charset=UTF-8:
+          schema:
+            $ref: '#/components/schemas/ErrorInfo'
+          examples:
+            ServiceUnavailable:
+              value:
+                status: 503
+                code: UNAVAILABLE
+                message: Service unavailable
+            Sorry_error:
+              value:
+                status: 503
+                code: UNAVAILABLE
+                message: sorry_error

```

I think we can remove this example, is not giving much information

```suggestion
            Sorry_error:
              value:
                status: 503
                code: UNAVAILABLE
                message: sorry_error

```